subsys/usb/class/bt_h4.c

```diff
@@ -217,7 +217,7 @@ static int bt_h4_vendor_handler(struct usb_setup_packet *setup,
 	    (setup->bRequest == 0x5c)) {
 		LOG_DBG("Device-to-Host, wLength %d, data %p",
 			setup->wLength, *data);
-		return 0;
+		/* Not yet supported */
```

Comment:
Didn't you miss `return -ENOTSUP;` here?

Reply:
My thought is that it is not necessary, it is only LOG_DBG and ends with final -ENOTSUP.

Reply:
Im wondering now, why is this
```
	if (REQTYPE_GET_DIR(setup->bmRequestType) == REQTYPE_DIR_TO_DEVICE &&
	    setup->bRequest == 0x5b) {
		LOG_DBG("Host-to-Device, data %p", *data);
		return 0;
```

valid in any way? Nothing happens here. Why returning 0? (few lines above).

Reply:
That was triage. Now, I think everything here should be -ENOTSUP, the line even higher also makes no sense. +1 to just return -ENOTSUP?